src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/FraudAlertModal.tsx

```diff
@@ -0,0 +1,139 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { useOverlayTriggerState } from "react-stately";
+import { useL10n } from "../../../../../../../hooks/l10n";
+import { useButton, useOverlayTrigger } from "react-aria";
+import { useRef } from "react";
+import { QuestionMarkCircle } from "../../../../../../../components/server/Icons";
+import { ModalOverlay } from "../../../../../../../components/client/dialog/ModalOverlay";
+import { Dialog } from "../../../../../../../components/client/dialog/Dialog";
+import Image from "next/image";
+import FraudAlertDialogIllustration from "../images/fraud-alert-modal-illustration.svg";
+import { Button } from "../../../../../../../components/server/Button";
+import styles from "./FraudAlertModal.module.scss";
+import Link from "next/link";
+
+export const FraudAlertModal = () => {
+  const l10n = useL10n();
+  const overlayTriggerState = useOverlayTriggerState({
+    defaultOpen: false,
+  });
+  const { triggerProps, overlayProps } = useOverlayTrigger(
+    { type: "dialog" },
+    overlayTriggerState
+  );
+  const triggerRef = useRef<HTMLButtonElement>(null);
+  const { buttonProps } = useButton(triggerProps, triggerRef);
+
+  const fraudAlertLink =
+    "https://consumer.ftc.gov/articles/what-know-about-credit-freezes-fraud-alerts";
+  const equifaxLink =
+    "https://www.equifax.com/personal/credit-report-services/";
+  const experianLink = "https://www.experian.com/help/";
+  const transunionLink = "https://www.transunion.com/credit-help";
+
+  return (
+    <>
+      <button
+        {...buttonProps}
+        ref={triggerRef}
+        className={styles.triggerButton}
+      >
+        <QuestionMarkCircle
+          alt={l10n.getString(
+            "fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-trigger-label"
+          )}
+          width={18}
+          height={18}
+        />
```

Comment:
Probably a minor issue, but the :question: now floats above the line, instead of being at the same height as the text, like the design has it:

![image](https://github.com/mozilla/blurts-server/assets/4251/8568d5e9-da14-4f49-aaf9-f8a8dbb8b529)